get-by-id-product-service/handler.js

```diff
@@ -1,31 +1,19 @@
 "use strict";
+const AWS = require('aws-sdk');
+AWS.config.update({ region: 'eu-west-1' });
+
+const dynamodb = new AWS.DynamoDB.DocumentClient();
 
 module.exports.getProductById = async (event) => {
     const { productId } = event.pathParameters;
-    const products = [
-        {
-            id: 1,
-            title: "Book1",
-            description: "The book is good",
-            price: 1004
-        },
-        {
-            id: 2,
-            title: "Book2",
-            description: "The book is sad",
-            price: 1005
-        },
-        {
-            id: 3,
-            title: "Book3",
-            description: "The book is nice",
-            price: 1005
-        },
-    ];
+    const products = await dynamodb.scan({ TableName: 'products', }).promise();
+    const stocks = await dynamodb.scan({ TableName: 'stocks', }).promise();
```

Comment:
I guess it would better to put this code inside try block below, since we can get an error here if something wrong happened with dynamo service.

Reply:
fixed